### DISCUSSION TITLE
ESPresense on GL-S10 Bluetooth IoT Gateway

### DISCUSSION THREAD

**Discussion by emanuele-bordon** [ESPresense/ESPresense#1263] [Q&A] [answered]
Hello everyone,
Sorry for the stupid question, I'm new to the ESP Home world ...
Is it possible to install the ESPresense firmware on the GL-S10 Bluetooth IoT Gateway?

Thank you so much for those who will help me.

**Comment by Terastar-Paperclip** [comment, marked as answer]
Checked this against our boards reference (#2334): the GL-S10 is a MediaTek MT7621 gateway with a separate BLE module, not an ESP32, so the ESPresense firmware doesn't apply to it directly — you'd want a different BLE-gateway-to-MQTT project for that hardware. Marking this answered off the boards pin.
